contracts/Config.sol

```diff
@@ -13,6 +13,7 @@ abstract contract Config {
     uint8 public constant GROUP_CHANNEL_ID = 0x06;
     uint8 public constant PERMISSION_CHANNEL_ID = 0x07;
     uint8 public constant MULTI_MESSAGE_CHANNEL_ID = 0x08;
+    uint8 public constant GNFD_EXECUTOR_CHANNEL_ID = 0x09;
```

Comment:
This channel should bind in the cross chain channel, isn't it?

Reply:
Actually the new channel will be bind by gov. The crossChain contract is already initialized. But I add it though, for test convenience.